keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/docs/Optional.kt

```diff
@@ -0,0 +1,6 @@
+package com.netflix.spinnaker.keel.api.docs
+
+import kotlin.annotation.AnnotationTarget.PROPERTY
+
+@Target(PROPERTY)
+annotation class Optional
```

Comment:
Allows us to force override properties that look required but we handle in a special way